Package.swift

```diff
@@ -26,6 +26,7 @@ let package = Package(
     .library(name: "Strings", targets: ["Strings"]),
     .library(name: "BraveVPN", targets: ["BraveVPN"]),
     .library(name: "BraveNews", targets: ["BraveNews"]),
+    .library(name: "BraveOnboarding", targets: ["BraveOnboarding"]),
```

Comment:
Can we name this just `Onboarding` or `AppOnboarding` something? It's not a Brave product like VPN/News/Wallet/Talk is.

Reply:
Fixed at https://github.com/brave/brave-ios/pull/6270/commits/1b5bbd88bd88e295e76cecf0c75da4fe09932d17